pkg/virt-handler/isolation/detector.go

```diff
@@ -159,6 +159,47 @@ func (s *socketBasedIsolationDetector) AdjustResources(vm *v1.VirtualMachineInst
 	return nil
 }
 
+const qemuProcessExecutable = "qemu-kvm"
```

Comment:
According to [this](https://github.com/kubevirt/kubevirt/blob/b2f3a6520ab821dbdf7a7a15f3da24bc32ed449a/cmd/virt-launcher/virt-launcher.go#L562) it can be `qemu-system` or `qemu-kvm`.

Reply:
Maybe @vladikr can shade a light on this. We do not deploy `qemu-system` on u/s, but maybe some other d/s is?
Even if they do, we can never assure it really works as we do not test it.

Reply:
yes, this can easily be `qemu-system`. our virt-launcher is based on an operating system that calls this executable qemu-kvm, but this can change.
Normally, we are looking for both, like in https://github.com/kubevirt/kubevirt/blob/master/cmd/virt-launcher/virt-launcher.go#L560
However, perhaps it's possible to look for a partial name `qemu` or make this configurable?

Reply:
@vladikr  We could set an array with the names of qemu process than search for one of them, as follows:
```
var qemuProcessExecutables = []string{"qemu-system", "qemu-kvm"}

func findIsolatedQemuProcess(processes []ps.Process, pid int) (ps.Process, error) {
        processes = childProcesses(processes, pid)
       for _, exec := range qemuProcessExecutables {
               if qemuProcess := lookupProcessByExecutable(processes, exec); qemuProcess != nil 
                       return qemuProcess, nil
               }
       }

       return nil, fmt.Errorf("no QEMU process found under process %d child processes", pid)
}
```
I think its safer than looking for `qemu-*`, there could be other qmeu processes other than `qemu-kvm/system`?
What you think?